recipes/patchelf/all/conanfile.py

```diff
@@ -1,5 +1,10 @@
-from conans import ConanFile, AutoToolsBuildEnvironment, tools
-from conans.errors import ConanInvalidConfiguration
+from conan import ConanFile
+from conan.tools.apple import is_apple_os
+from conan.tools.files import get, chdir, rmdir
+from conan.tools.scm import Version
+from conans import AutoToolsBuildEnvironment
+from conan.errors import ConanInvalidConfiguration
+import conans
 import os
 
 required_conan_version = ">=1.33.0"
```

Comment:
```suggestion
required_conan_version = ">=1.51.3"
```
is_apple_os requires 1.51